node_modules/oae-content/lib/internal/util.js

```diff
@@ -104,7 +108,45 @@ var augmentContent = module.exports.augmentContent = function(ctx, content, dura
             content.previews.wideUrl = getSignedDownloadUrl(ctx, content.previews.wideUri, duration, offset);
             delete content.previews.wideUri;
         }
+
+    }
+};
+
+/**
+ * Generate a signed url that can be used to show video previews in the browser for a video file stored in AWS S3
+ *
+ * @param  {Context}    ctx               Current request context
+ * @param  {String}     content           The content object that needs a signed S3 video URL
+ * @param  {Function}   callback          Standard callback function
+ * @param  {Object}     callback.err      An error that occurred, if any
+ */
+var getSignedVideoUrl = module.exports.getSignedVideoUrl = function(ctx, content, callback) {
+    if (!content.previews.videoKey) {
+        return callback();
     }
+
+    var key = content.previews.videoKey;
```

Comment:
Rather than access `content.previews.videoKey` twice, we could first do `var key = content.previews.videoKey;` and then check `if (!key) {...` afterward.